include/spatio_temporal_voxel_layer/spatio_temporal_voxel_layer.hpp

```diff
@@ -173,6 +173,8 @@ class SpatioTemporalVoxelLayer : public nav2_costmap_2d::CostmapLayer
   std::vector<observation::MeasurementReading> _static_observations;
   std::unique_ptr<volume_grid::SpatioTemporalVoxelGrid> _voxel_grid;
   boost::recursive_mutex _voxel_grid_lock;
+  rclcpp::Clock::SharedPtr clock;
+  rclcpp::Logger logger{rclcpp::get_logger("spatio_temporal_voxel_layer")};
```

Comment:
Don't initialize here

Reply:
`logger` is the one you initialized, so remove that. Also logger is missing the `_`.